README.md

```diff
@@ -39,3 +39,5 @@ See [CONTRIBUTING.md](./CONTRIBUTING.md) for more information.
 [4]: https://kubernetes.io/docs/concepts/storage/storage-classes/#the-storageclass-resource
 [5]: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#expanding-persistent-volumes-claims
 [6]: https://kubernetes.io/docs/concepts/storage/volume-snapshots
+
+**&ast;**: Externally created volumes [must have the Disc Serial Exposure flag enabled](https://api.ionos.com/docs/cloud/v6/#tag/Volumes/operation/datacentersVolumesPatch), otherwise the driver won't be able to identify them during the mounting process.
```

Comment:
```suggestion
**&ast;**Externally created volumes [must have the Disc Serial Exposure flag enabled](https://api.ionos.com/docs/cloud/v6/#tag/Volumes/operation/datacentersVolumesPatch), otherwise the driver won't be able to identify them during the mounting process.
```